communities/activity-event-and-media-recommendations.md

```diff
@@ -0,0 +1,184 @@
+1. Two tech-related pieces of media that interest us
+
+## Preparation
+
+There should be two empty slide decks prepared:
+
+1. One slide deck for everyone to put their event recommendations
+1. One slide deck for everyone to put their media recommendations
+
+**The slide decks should be configured to auto-advance every thirty seconds**.
+
+### !callout-warning
+
+## Presentations Will Be Short and Low-Stakes
+
+The slide decks should be configured to auto-advance every thirty seconds.
+
+### !end-callout
+
+## Activity Instructions
+
+Spend around 25 minutes finding recommendations and creating slides for them:
+
+1. Follow the directions below to find one recommended event, and create one slide for it.
+1. Follow the directions below to find two recommended works of media, and create one slide for each recommendation.
+
+Follow the facilitator's directions for the presentation.
+
+- When any of your slides comes up, describe what the recommendation is, and why you recommend it.
+- You may pass your turn by saying "pass."
+- You may end your turn early by saying "pass."
+- You will have a **maximum of 30 seconds per slide**.
+
+## Event Recommendation Criteria
+
+Your goal is to find something **where you can meet more developers that share something with you**.
+
+You can recommend any type of event! Events can vary by type, subject, location, and presence.
+
+| Type           | Subject                     | Location | Presence  |
+| -------------- | --------------------------- | -------- | --------- |
+| Meet-ups       | Identity                    | Local    | Digital   |
+| Conferences    | Special Interest            | National | In-Person |
+| Support Groups | Identity & Special Interest | Global   |
+| Hackathons     |                             |          |
+
+Examples:
+
+- Atlanta Mobile Developers Group Meet-up
+- Game Devs of Color Expo
+- Puget Sound Programming Python (PuPPy)
+- MozFest, an art and tech meetup/maker festival/gathering for activists
+- Advent of Code
+- Lesbians Who Tech (Not IRL) Pride Summit
+- Seattle Women in Machine Learning and Data Science
+- Deconstruct Conf
+- TechTogether Seattle
+
+If you're a member of the following groups, or another identity group not listed here, we encourage you to find an event that centers that experience:
+
+- Black
+- Indigenous
+- POC
+- LGBTQIA+
+- Trans
+- Disabled
+- Neurodivergent
+
+### !callout-info
+
+## Getting Started
+
+If you're unsure where to begin research, we can search the Internet for the phrase `"${INTEREST} ${CITY} meetup"`, where `INTEREST` is an interest of yours and `CITY` is the name of the city you live in. For example, we could search `"software developer seattle meetup"` or `"data science PNW meetup"`.
+
+### !end-callout
+
+### Event Slide Requirements
+
+- Your name and pronouns
+- A link to the event
+- A graphic/photograph/picture/logo
+- Answer this question: How could this event contribute to your career interests?
```

Comment:
personal perspective for the question (similar to recommendation from other review)
```suggestion
- Answer this question: How could this event contribute to my career interests?
```